spec/factories/grda_warehouse/data_sources.rb

```diff
@@ -54,4 +54,12 @@
     source_type { :sftp }
     visible_in_window { true }
   end
+
+  factory :health_data_source, class: 'GrdaWarehouse::DataSource' do
+    name { 'Health' }
+    short_name { 'Health' }
+    # association :client, factory: :grda_warehouse_hud_client
+    source_type { :sftp }
```

Comment:
I think this should be `source_type { nil }` as those wouldn't ever be imported as HMIS records.